http/service.go

```diff
@@ -1256,10 +1256,11 @@ func (s *Service) handleQuery(w http.ResponseWriter, r *http.Request, qp QueryPa
 			DbTimeout:   int64(qp.DBTimeout(0)),
 			Statements:  queries,
 		},
-		Timings:         qp.Timings(),
-		Level:           qp.Level(),
-		Freshness:       qp.Freshness().Nanoseconds(),
-		FreshnessStrict: qp.FreshnessStrict(),
+		Timings:                      qp.Timings(),
+		Level:                        qp.Level(),
+		Freshness:                    qp.Freshness().Nanoseconds(),
+		FreshnessStrict:              qp.FreshnessStrict(),
+		EnableLeaderReadOptimization: qp.EnableLeaderReadOptimization(),
```

Comment:
Let's call this field `EnableLeaderRO`. Verbose variable names is not Go style.

Reply:
`EnableLRO`

Reply:
OK, well let's just do `LeaderReadOpt` -- that's what we should go with. There is no need for "Enable" since it's a boolean.